### ISSUE TITLE
Trying to execute a sync operation with a storage manager for an anonymous account

### ISSUE THREAD

**Issue by araujo-marcio** [nextcloud/android#13293]
### Cause of error
```java
Exception in thread "Thread-7" java.lang.IllegalArgumentException: Trying to execute a sync operation with a storage manager for an anonymous account
    at com.owncloud.android.operations.common.SyncOperation.execute(SyncOperation.java:49)
    at com.owncloud.android.ui.activity.DrawerActivity.lambda$fetchExternalLinks$9(DrawerActivity.java:1231)
    at com.owncloud.android.ui.activity.DrawerActivity.$r8$lambda$oHM2j1MZkviAIdyIFWoI2kSeF9c(Unknown Source:0)
    at com.owncloud.android.ui.activity.DrawerActivity$$ExternalSyntheticLambda6.run(D8$$SyntheticClass:0)
    at java.lang.Thread.run(Thread.java:1012)

```

### App information
* ID: `com.nextcloud.client`
* Version: `30290290`
* Build flavor: `gplay`

### Device information
* Brand: `POCO`
* Device: `ingres`
* Model: `21121210G`
* Id: `UKQ1.230917.001`
* Product: `ingres_global`

### Firmware
* SDK: `34`
* Release: `14`
* Incremental: `V816.0.3.0.ULJMIXM`

**Comment by alperozturk96** [comment]
@araujo-marcio Can you be able to launch app and see the file list or app is crashing immediately?

Also, could you try with [3.29.1](https://github.com/nextcloud/android/releases/tag/stable-3.29.1)?

**Comment by joshtrichards** [comment]
Duplicate of #13323

**Comment by joshtrichards** [comment]
Presumed fixed in #13444
